lib/cheffish/merged_config.rb

```diff
@@ -3,8 +3,8 @@
 module Cheffish
   class MergedConfig
     def initialize(*configs)
-      @configs = configs.map { |config| Chef::Mash.from_hash config }
-      @merge_arrays = Chef::Mash.new
+      @configs = configs.map { |config| ::Mash.from_hash config }
+      @merge_arrays = ::Mash.new
```

Comment:
should probably continue to use Chef::Mash?  i think we do overwrite ::Mash but i think its safer to use Chef::Mash and we should probably stop using ::Mash since someone else might scribble over that as well...